src/main/java/edu/hawaii/its/api/configuration/OotbGrouperPropertyConfigurer.java

```diff
@@ -40,9 +42,13 @@
 @ActiveProfiles("ootb")
 class OotbGrouperPropertyConfigurer {
 
-    private PropertyLocator propertyLocator = new PropertyLocator("src/main/resources", "data.harness.properties");
+    private static final Log log = LogFactory.getLog(OotbGrouperPropertyConfigurer.class);
 
-    public static final Log log = LogFactory.getLog(OotbGrouperPropertyConfigurer.class);
+    private final PropertyLocator propertyLocator;
+
+    public OotbGrouperPropertyConfigurer() {
+        this.propertyLocator = new PropertyLocator("src/main/resources", "data.harness.properties");
+    }
```

Comment:
I would move this back to how it was before since it was not previously being autowired.